### DISCUSSION TITLE
student developer programme

### DISCUSSION THREAD

**Discussion by Vishali3261** [community/community#131916] [GitHub Education]
### Hi everyone,

my student developer programme application is verified but but i did not get benefit till now

**Comment by paiineli** [comment]
Hi,

Here are a few steps you can try:

Check Your Email: Ensure you received any confirmation regarding your benefits. Sometimes, additional steps may be required.

Log Out and Log In: Try logging out of your GitHub account and then logging back in to refresh your account status.

Verify Your Account: Double-check that your educational email is properly linked and verified in your GitHub account settings.

Contact GitHub Support: If you still don’t see the benefits, I recommend reaching out to GitHub Support. They can provide specific assistance regarding your account.

I hope this issue gets resolved quickly!

Best,

**Comment by samus-aran** [comment]
I'll be closing this discussion due to lack of recent activity. We can confirm that you have since been successful - enjoy your journey through GitHub Education.
